src/HasStateTransitions.php

```diff
@@ -47,7 +47,7 @@ public function transitionStateTo(TransitionState $to, string $field = null): se
     /**
      * @return array<\LenderSpender\StateTransitionWorkflow\TransitionState>
      */
-    public function getAvailableStateTransitions(?string $field = null): array
+    public function getAvailableStateTransitions(string $field = null): array
```

Comment:
I've seen you remove the nullable type on multiple occasions with a `null` default value. Do you have a specific reason to do so?

Reply:
Didn't know I did this, apparently my phpcs does this.